staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/testdata/README.md

```diff
@@ -0,0 +1,4 @@
+`swagger.json` contains static openapi definitions needed to build models for custom resource types.
+
+It was created by removing all path and non-objectmeta/int-or-string definitions from 
```

Comment:
I'm not entirely sure I understand that?

Reply:
started with `api/openapi-spec/swagger.json` and removed all the `.paths` bits, and all the `.definitions` that weren't related to int-or-string or objectmeta types